api/employee/employeeRouter.js

```diff
@@ -0,0 +1,143 @@
+const router = require('express').Router();
+const emp = require('./employeeModel.js');
+const userModel = require('../user/userModel.js');
+
+const auth = require('../../middleware/authMiddleWare');
+
+router.use(auth.validateId);
+
+// Get all employees from the database
+router.get('/', (req, res) => {
```

Comment:
we could get the org id from req.profile and find all employees for a single organization

Reply:
Done.